java/RSPEC-1764.java

```diff
@@ -0,0 +1,14 @@
+class IdentExprBinary {
+    void test() {
+        int a = 1;
+        int b = 2;
+        if (a == a) {}
```

Comment:
RSPEC-1764: Equal terms on both sides of binary expression, 
<!-- lookout footnote separator -->
_Comment made by the analyzer sonarcheck_analyzer._